src/form/Form.jsx

```diff
@@ -164,54 +263,90 @@ export default class Form extends React.Component {
       pristine: true,
       submitting: false
     })
-  }
+  };
+
+  getFormFieldsStructure = () => {
+    if (this.props.value && this.props.value.size > 0) {
+      return [...this.props.value.keys()]
```

Comment:
1. Does this mean it doesn't work for nested structures?
2. Do nested structures exist?

Reply:
1. yes, it looks so

Reply:
ok then